idaes/models_extra/column_models/condenser.py

```diff
@@ -243,9 +243,28 @@ def build(self):
 
         self._make_ports()
 
+        def rule_reflux(self):
+            return self.reflux_ratio * (1 + self.reflux_ratio) ** -1
+
+        self.expr1 = Expression(rule=rule_reflux)
```

Comment:
I think this needs a better name. Also note that you can do `@self.Expression` in the same way as `Constraints`.